plugin/trino-teradata/src/test/java/io/trino/plugin/teradata/clearscapeintegrations/ClearScapeEnvVariables.java

```diff
@@ -0,0 +1,17 @@
+package io.trino.plugin.teradata.clearscapeintegrations;
+
+public class ClearScapeEnvVariables
+{
+    private ClearScapeEnvVariables()
+    {
+        // Utility class - prevent instantiation
+    }
+
+//    public static final String ENV_CLEARSCAPE_ENABLED = "test.clearscape.enabled";
+    public static final String ENV_CLEARSCAPE_URL = "https://api.clearscape.teradata.com";
+    public static final String ENV_CLEARSAOPE_USERNAME = "demo_user";
+    public static final String ENV_CLEARSCAPE_PASSWORD = "Devtools@007";
+    public static final String ENV_CLEARSCAPE_NAME = "teradata-trino";
+    public static final String ENV_CLEARSCAPE_TOKEN = "Uy5o531CkA4UzPchLs7xHx59KdDWVZTFnlDrcq7nySXu6K56yKHGMICeevt2QaLK";
```

Comment:
Remove this token. Going forward dont push the sensitive information to github. Please send an email to clearscape team to change this token.